api/insight/converter.go

```diff
@@ -6,57 +6,60 @@ package insight
 
 import (
 	"github.com/decred/dcrd/blockchain"
-	"github.com/decred/dcrd/chaincfg"
 	"github.com/decred/dcrd/dcrjson"
 	"github.com/decred/dcrd/dcrutil"
 	apitypes "github.com/decred/dcrdata/api/types"
 )
 
 // TxConverter converts dcrd-tx to insight tx
 func (c *insightApiContext) TxConverter(txs []*dcrjson.TxRawResult) ([]apitypes.InsightTx, error) {
-	return c.TxConverterWithParams(txs, false, false, false)
+	return c.DcrToInsightTxns(txs, false, false, false)
 }
 
-// TxConverterWithParams takes struct with filter params
-func (c *insightApiContext) TxConverterWithParams(txs []*dcrjson.TxRawResult, noAsm bool, noScriptSig bool, noSpent bool) ([]apitypes.InsightTx, error) {
-	newTxs := []apitypes.InsightTx{}
+// DcrToInsightTxns takes struct with filter params
+func (c *insightApiContext) DcrToInsightTxns(txs []*dcrjson.TxRawResult,
+	noAsm, noScriptSig, noSpent bool) ([]apitypes.InsightTx, error) {
+	var newTxs []apitypes.InsightTx
 	for _, tx := range txs {
 
-		vInSum := float64(0)
```

Comment:
`var vInSum float64` inits the variable to the zero value for the data type

Reply:
Yea, i'm still unclear when you want me to use := vs var inits.  we are getting there :)

Reply:
Well same here, we're not super strict, but we try to be internally consistent.

A couple resources for idiomatic go:
https://golang.org/doc/effective_go.html?
https://dmitri.shuralyov.com/idiomatic-go

BTW, this was one of my favorite pages when I was first learning go:
http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/